wire-ios/Wire-iOS/Sources/Analytics/Analytics.swift

```diff
@@ -83,6 +82,7 @@ final class Analytics: NSObject {
 
         attributes.merge(userInfo, strategy: .preferNew)
         tagEvent("e2ee.failed_message_decryption", attributes: attributes)
+        logger.info("e2ee.failed_message_decryption \(attributes)")
```

Comment:
I wouldn't add the log here, see my other comment

Reply:
ok, updated logs on the calling place

Reply:
We should still remove the changes in this file.

Reply:
removed